protostar/cheatable_starknet/controllers/io.py

```diff
@@ -0,0 +1,12 @@
+from protostar.starknet.data_transformer import CairoData
+from protostar.cairo.short_string import short_string_to_str
+
+
+class IOController:
+    @staticmethod
+    def protostar_print(data: CairoData):
+        for data_item in data:
+            str_data = short_string_to_str(data_item)
+            original_value_msg = "original value: [" + str(data_item) + "]"
+            converted_value_msg = "converted to a string: [" + str_data + "]"
```

Comment:
I think using f string to construct this would be more readable.

Reply:
[f string](https://github.com/software-mansion/protostar/pull/1803/commits/35a863a06873ee9a0ceb0c159a5b1ed797ac9108)